Source/WebKit/UIProcess/RemoteLayerTree/mac/RemoteLayerTreeEventDispatcher.mm

```diff
@@ -204,8 +204,8 @@ void displayLinkFired(PlatformDisplayID displayID, DisplayUpdate, bool wantsFull
 
     willHandleWheelEvent(wheelEvent);
 
-    ScrollingThread::dispatch([dispatcher = Ref { *this }, wheelEvent, rubberBandableEdges] {
-        dispatcher->scrollingThreadHandleWheelEvent(wheelEvent, rubberBandableEdges);
+    ScrollingThread::dispatch([dispatcher = Ref { *this }, wheelEvent = WebWheelEvent { wheelEvent }, rubberBandableEdges] {
+        dispatcher->scrollingThreadHandleWheelEvent(CheckedRef { wheelEvent }.get(), rubberBandableEdges);
```

Comment:
Why do we need to do this change?
AIUI, wheelEvent is captured by value so is safe for the lifetime of the lambda.

Reply:
https://build.webkit.org/results/Apple-Sequoia-Safer-CPP-Checks/294329%40main%20%283257%29/scan-build-output/StaticAnalyzer/WebKit/StaticAnalyzerReports/report-RemoteLayerTreeEventDispatcher.mm-unknown-0-ddd2d4.html#EndPath